eng/cmake/global_compile_options.txt

```diff
@@ -17,10 +17,12 @@ elseif(CMAKE_C_COMPILER_ID MATCHES "Clang")
   endif()
 
   add_compile_options(-Xclang -Wall -Wextra -pedantic  ${WARNINGS_AS_ERRORS_FLAG} -Wdocumentation -Wdocumentation-unknown-command -fcomment-block-commands=retval -Wcast-qual -Wunused -Wuninitialized -Wmissing-declarations -Wconversion -Wpointer-arith -Wshadow -Wfloat-equal)
-else()
+elseif(CMAKE_C_COMPILER_ID MATCHES "GCC")
```

Comment:
replace GCC  with GNU

Reply:
Ah you're right my apologies
https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html